src/layer/lineLayer.js

```diff
@@ -0,0 +1,42 @@
+import { Observable } from '../util/observable';
+import { Line as RenderableLine } from '../renderable/line';
+import { LineLayerRender } from '../render/lineLayerRender';
+import { LineMesh } from '../mesh/lineMesh';
+
+export class LineLayer extends Observable {
```

Comment:
LineLayer is one of the VectorLayer.  We will add TextLayer, IconLayer, CircleLayer and others.